test/adapters/code-highlight-adapter.test.ts

```diff
@@ -50,8 +50,10 @@ describe('CodeHighlightAdapter', () => {
       expect(highlightStub.called).to.be.true;
 
       invariant(result != null);
-      expect(result.length).to.equal(1);
-      expect(result[0].isEqual(new Range([0, 1], [0, 2]))).to.be.true;
+      if (result) {
```

Comment:
Tests required a bunch of null/undefined checks so that the compiler would be happy